doc/sphinx/inter.rst

```diff
@@ -1806,36 +1806,27 @@ properties.
 Scafacos
 ~~~~~~~~
 
-Espresso can use the electrostatics methods from the Scafacos *Scalable
-fast Coulomb solvers* library.
+Espresso can use the electrostatics methods from the SCAFACOS *Scalable
+fast Coulomb solvers* library. The specific methods available depend on the compile-time options of the library, and can be queried using :attr:`espressomd.scafacos.available\_methods()`
 
-scafacos\_methods
+To use SCAFACOS, create an instance of :attr:`espressomd.electrostatics.Scafacos` and add it to the list of active actors. Three parameters have to be specified:
+* method_name: name of the SCAFACOS method being used.
+* method_params: dictionary containing the method-specific parameters
+* bjerrum_length
+The method-specific parameters are described in the SCAFACOS manual.
+Additionally, methods supporting tuning have the parameter ``tolerance_field`` which sets the desired root mean square accuracy for the electric field 
 
-scafacos.available\_methods()
-
-This shows the methods available at the compile time of . Scafacos can
-be used as Coulomb solver for the system.
-
-inter coulomb scafacos
-
-[ ]
-
-Here is a scafacos method as returned by ``scafacos_methods``.
-``tolerance_field`` sets the desisired rms accuracy for the electric
-field if supported by the method. is a list of parameters as described
-by the Scafacos manual. If parameters of the solver are not set, and the
-method supports it, the open parameteres are tuned. To use the ``ewald``
-solver from scafacos as electrostatics solver for your system, set its
+To use the, e.g.,  ``ewald`` solver from SCAFACOS as electrostatics solver for your system, set its
 cutoff to :math:`1.5` and tune the other parameters for an accuracy of
-:math:`10^{-3}`, use the command
-
-inter coulomb 1.0 scafacos ewald ewald\_r\_cut 1.5 tolerance\_field 1e-3
+:math:`10^{-3}`, use::
+  from espressomd.electrostatics import Scafacos
+  scafacos=Scafacos(bjerrum_length=1,method_name="ewald", 
+    method_params={"ewald\_r\_cut":1.5, "tolerance\_field":1e-3})
```

Comment:
The backslashes matter here and are printed as is.  They are a relic from the LaTeX docs and should be remove in favour of literal underscores.